blueocean-pipeline-editor/src/main/less/editor.less

```diff
@@ -769,6 +712,40 @@
         line-height: 17px;
         font-weight: bold;
     }
+
+    .name-new-parallel.Dialog {
+        .TextInput {
+            display: block;
+            width: 100%;
+        }
+    }
+
+    // collapse-height-enter collapse-height-enter-active
+    .collapse-height(@max-height: 300px) {
+        overflow: hidden;
+        &-enter {
```

Comment:
Nice, I actually didn't know about this syntax. 👍

Reply:
I actually did something that @cliffmeyers  didn't know about?!? that is the MEGA-:bee: for the week :)

Reply:
Worthy of a lucite trophy?